Mathlib/MeasureTheory/Measure/Haar/Unique.lean

```diff
@@ -994,6 +994,42 @@ theorem MeasurePreserving.zpow [CompactSpace G] [RootableBy G ℤ]
 
 end CommGroup
 
+section DistribMulAction
+variable {G A : Type*} [Group G] [AddCommGroup A] [DistribMulAction G A] [MeasurableSpace A]
+  -- We only need `MeasurableConstSMul G A` but we don't have this class. So we erroneously must
+  -- assume `MeasurableSpace G` + `MeasurableSMul G A`
```

Comment:
Is this the comment you wanted to make here?

Reply:
Yes, absolutely. Why do you think it is wrong?

Reply:
I will merge now to press on. Can always fix later

Reply:
Since you have `BorelSpace A` and the action of `G`  is continuous, isn't ``MeasurableConstSMul G A` morally true here, meaning you shouldn't need any additional typeclass?